roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -342,6 +342,16 @@
     ports:
     - port: 9300
 
+- name: Modify logging-{{ es_component }}-cluster service to be headless
```

Comment:
Why would we not just adjust the service object when we create it? it seems like an extra step to later edit it

Reply:
There is no way to set the 'spec.publish...' without modifying 'oc_service' which for sake of convenience (Impatience?) am not inclined to take on.